### PR TITLE
first version of Blender to glTF page

### PR DESCRIPTION
**Pull request by Vinc3r**
#556

### CODE REVIEW

**Comment by Vinc3r** [comment]
notice the wink: screenshot is on Edge :-p

File: content/resources/Exporter/Blender_to_glTF.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+<body>
+    <canvas id="canvas"></canvas>
+    <script type="text/javascript">
+        var canvas = document.getElementById("canvas");
+        var engine = new BABYLON.Engine(canvas, true);
+        
+        // here the doc for Load function: http://doc.babylonjs.com/api/classes/babylon.sceneloader#load
+        BABYLON.SceneLoader.Load("", "02_suzanne.gltf", engine, function (scene) {
+		
+            //BJS seens to not import - yet - glTF cameras, so we have to manually create one
```
Comment (deltakosh) [inline content/resources/Exporter/Blender_to_glTF.md]: We can remove this one now:) it is fixed

**Review by deltakosh — CHANGES_REQUESTED** [review]

**Review by deltakosh — APPROVED** [review]

File: content/resources/Exporter/Blender_to_glTF.md (hunk 1 above)
Comment (Vinc3r) [reply content/resources/Exporter/Blender_to_glTF.md]: I remove the comment, but after few tries I wasn't be able to get my camera from gtlf (using directly ``` scene.activeCamera.attachControl(canvas) ``` or even ``` scene.cameras[0].attachControl(canvas) ```. So for now I left the camera creation, I think maybe the js babylon file at  ``` https://preview.babylonjs.com/loaders/babylonjs.loaders.js ``` is not yet updated?

Comment (deltakosh) [reply content/resources/Exporter/Blender_to_glTF.md]: Sorry I forgot to mention that the PR is not yet merged :)

**Pull request merged**